hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/StorageType.java

```diff
@@ -34,28 +34,35 @@
 @InterfaceStability.Unstable
 public enum StorageType {
   // sorted by the speed of the storage types, from fast to slow
-  RAM_DISK(true),
-  SSD(false),
-  DISK(false),
-  ARCHIVE(false),
-  PROVIDED(false);
+  RAM_DISK(true, true),
+  NVDIMM(false, true),
```

Comment:
Let's add only this line

Reply:
Sorry, I don't understand.

Reply:
As we are adding only "NVDIMM",So I expect one line change here but I missed that you added one more param to enum.